core/templates/domain/exploration/LostChangeObjectFactory.ts

```diff
@@ -55,8 +55,10 @@ export interface LostChangeBackendDict {
   'new_state_name'?: string;
   'old_state_name'?: string;
   'state_name'?: string;
-  'new_value'?: LostChangeValue;
-  'old_value'?: LostChangeValue;
+  // 'new_value' is null when an Exploration is deleted or Removed.
+  'new_value'?: LostChangeValue | null;
```

Comment:
'new_value' can be null
https://github.com/oppia/oppia/blob/26b1b5dad4b42a540b2557aba1d27b7bd099f190/core/domain/suggestion_services_test.py#L1823-L1831